src/engine/effects/engineeffectchain.cpp

```diff
@@ -165,98 +165,77 @@ void EngineEffectChain::process(const ChannelHandle& handle,
 
     EffectProcessor::EnableState effectiveEnableState = channel_info.enable_state;
 
+    if (channel_info.enable_state == EffectProcessor::DISABLING) {
+        channel_info.enable_state = EffectProcessor::DISABLED;
```

Comment:
The issue was discussed almost completely here: 
https://bugs.launchpad.net/mixxx/+bug/1481170